README.md

```diff
@@ -8,21 +8,55 @@ It can currently repair violations of [15+ rules](/docs/HANDLED_RULES.md).
 
 A JDK (Java 11+)
 
-### Usage
-
-##### If you want to run it from source code
+### Build
 
 1) Clone this repository: `git clone https://github.com/SpoonLabs/sorald.git`
 
-2) Build and run the tool:
+2) Build:
 
  ```bash
 $ cd sorald
 $ mvn package -DskipTests
-$ java -jar target/sorald-1.1-SNAPSHOT-jar-with-dependencies.jar <arguments>
  ```
 
-The arguments are the following:
+The Sorald application can now be found in
+`target/sorald-1.1-SNAPSHOT-jar-with-dependencies.jar`.
+
+### Usage
+
+Sorald can do two things: automatically repair violations of Sonar rules in a
```

Comment:
"Sorald can do two things" -> "Sorald can do two tasks"

Reply:
How about this ?

```suggestion
Sorald can perform two different tasks: automatically repair violations of Sonar rules in a
```

Reply:
Perfect!

Reply:
Done!